Formula/python@3.9.rb

```diff
@@ -128,7 +122,11 @@ def install
       cflags << "-I#{MacOS.sdk_path}/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers"
     end
     # Avoid linking to libgcc https://mail.python.org/pipermail/python-dev/2012-February/116205.html
-    args << "MACOSX_DEPLOYMENT_TARGET=#{MacOS.version}"
+    if MacOS.version >= :big_sur
+      args << "MACOSX_DEPLOYMENT_TARGET=11.0.1"
+    else
+      args << "MACOSX_DEPLOYMENT_TARGET=#{MacOS.version}"
+    end
```

Comment:
This should fix the style issue

```suggestion
    args << if MacOS.version >= :big_sur
      "MACOSX_DEPLOYMENT_TARGET=11.0.1"
    else
      "MACOSX_DEPLOYMENT_TARGET=#{MacOS.version}"
    end
```